src/asynch.rs

```diff
@@ -6,7 +6,8 @@
 //! [`futures::executor`]: https://docs.rs/futures/0.3.13/futures/executor/index.html
 
 use crate::runtime::execution::ExecutionState;
-use crate::runtime::task::{TaskId, TaskType};
+use crate::runtime::task::PendingTask;
+use crate::runtime::task::TaskId;
```

Comment:
merge these into one line (your IDE might be generating them this way by default)